Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -22,17 +22,24 @@
 #import <UIKit/UIActivityIndicatorView.h>
 #import <UIKit/UIColor.h>
 
-
-#import "CppWinRTHelpers.h"
 #import "UIViewInternal.h"
+#import "XamlControls.h"
 #import "XamlUtilities.h"
+#import "CppWinRTHelpers.h"
 
 #include "COMIncludes.h"
 #import <winrt/Windows.UI.Xaml.Controls.h>
 #include "COMIncludes_End.h"
 
+// This color matches what iOS uses internally for the gray style.
+// NOTE: UIColor.grayColor differs slightly from this. Its components are W:0.5, A:1.0
+#define activityIndicatorStyleGrayColor = [UIColor colorWithWhite:0.0 alpha:0.45]
```

Comment:
Okay, so kill the define and just inline that where it's used? Yi had commented on the original review I submitted suggesting I add a constant for the color components to make it more clear, so I was trying to think of a way to make the entire color definition more clear since this is a special case.